src/main/kotlin/uk/gov/justice/hmpps/architecture/model/UseOfForce.kt

```diff
@@ -0,0 +1,131 @@
+package uk.gov.justice.hmpps.architecture.model
+
+import com.structurizr.model.Container
+import com.structurizr.model.Model
+import com.structurizr.model.Person
+import com.structurizr.model.SoftwareSystem
+import com.structurizr.view.AutomaticLayout
+import com.structurizr.view.ViewSet
+import uk.gov.justice.hmpps.architecture.HMPPSSoftwareSystem
+import uk.gov.justice.hmpps.architecture.annotations.Notifier
+import uk.gov.justice.hmpps.architecture.annotations.Tags
+
+class UseOfForce private constructor() {
+
+  companion object : HMPPSSoftwareSystem {
+    lateinit var model: Model
+    lateinit var system: SoftwareSystem
+    lateinit var useOfForceService: Container
+    lateinit var db: Container
+    lateinit var redis: Container
+
+    lateinit var reportCreator: Person
+    lateinit var coordinator: Person
+    lateinit var reviewer: Person
+    lateinit var involvedStaff: Person
+
+    override fun defineModelEntities(model: Model) {
+      this.model = model
+
+      system = model.addSoftwareSystem(
+        "Use of Force",
+        "A system for allowing prison staff to report a Use of Force instance",
+      ).apply {
+        val PRISON_PROBATION_PROPERTY_NAME = "business_unit"
+        val PRISON_SERVICE = "prisons"
+        addProperty(PRISON_PROBATION_PROPERTY_NAME, PRISON_SERVICE)
+      }
+
+      db = system.addContainer(
+        "Use of Force Database",
+        "Database to store Use of Force reports", "RDS Postgres DB"
+      ).apply {
+        Tags.DATABASE.addTo(this)
+        CloudPlatform.rds.add(this)
+      }
+
+      redis = system.addContainer(
+        "In-memory data store",
+        "handles processing queues for email distribution and NOMIS movements",
+        "REDIS"
+      ).apply {
+        Tags.SOFTWARE_AS_A_SERVICE.addTo(this)
+        CloudPlatform.elasticache.add(this)
+      }
+
+      useOfForceService = system.addContainer("Use of Force service", "Allows prison staff to report a Use of Force instance", "Node").apply {
+        setUrl("https://github.com/ministryofjustice/use-of-force")
+
+        CloudPlatform.kubernetes.add(this)
+      }
+
+      reportCreator = model.addPerson("Report creator user", "Prison staff who creates reports and provides statements").apply {
+        uses(useOfForceService, "Creates Use of Force reports")
+      }
+
+      coordinator = model.addPerson("Coordinator user", "Prison staff who can view all complete and in progress reports and statements across their caseload").apply {
+        uses(useOfForceService, "Views Use of Force reports")
```

Comment:
_not a blocker, suggestion_
It's usually useful for posterity to also state the intent of these users, eg. "views these reports to {some business purpose}".

Reply:
Thank you - I'll have a think about this